bin/epispot

```diff
@@ -0,0 +1,74 @@
+"""Shell Initialization"""
+import pathlib
+import epispot
+
+try:
+    import fire
+except:
+    print("Please install Fire to use epispot's CLI")
+    print('>>> pip install fire')
+
```

Comment:
Add \n per PEP8 convention: 2 blank lines before class definition
```suggestion


```

Reply:
Fixed in commit 95c948507e1f3479508132612195d9f40f7e3dfc